crates/typst/src/geom/axes.rs

```diff
@@ -178,14 +178,19 @@ impl Axis {
             Self::Y => Self::X,
         }
     }
+
+    /// A description of this axis' direction.
+    pub fn description(self) -> &'static str {
+        match self {
+            Self::X => "horizontal",
+            Self::Y => "vertical",
+        }
+    }
```

Comment:
I use this method for all `.axis()` methods on alignment. Thus, they return `"horizontal"` and `"vertical"` (just like the `repr` of axis would, although axis isn't really a Typst value anyway, so that doesn't matter much). Feel free to suggest changing this to `"x"` or `"y"` if you prefer.